### DISCUSSION TITLE
Videos appear as link

### DISCUSSION THREAD

**Discussion by vazome** [community/community#17506] [Mobile] [answered]
Here is an example on IOS:
![image](https://user-images.githubusercontent.com/46573198/170995539-119f8a6e-0827-454d-8fc1-a555e4a9e3ad.png)

**Comment by HenSquared** [comment, marked as answer]
Hi @vazome,

Thanks for submitting this feedback! I wasn't able to reproduce this issue on a PR with a video in it. Do you mind sharing a link to an issue or PR where you notice the videos are rendered as links instead of videos?

Thank you.

**Reply by vazome** [reply]
Hi @HenSquared!

It is in project video:
https://github.com/vazome/price-grabber

**Reply by HenSquared** [reply]
Thanks for this repo link. 

I see this issue now too. I've added repro steps and this link to an Issue to address this bug.
